### DISCUSSION TITLE
Scheduled reminders: reminders for people in child teams

### DISCUSSION THREAD

**Discussion by glasserc** [community/community#10871] [Pull Requests]
At my employer, we have parent teams which contain child teams. From time to time it's appropriate to tag the parent team for review of a PR rather than identifying each child team. (Each team is busy working on features, but systemic changes might require notifying a whole department.) The "scheduled pull reminders" integration documented at https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/managing-your-scheduled-reminders seems to only notify the people who are directly assigned to that team, not those assigned to child teams. This differs from the `review-requested/@me` filter on the PR page, which shows PRs that are assigned to parent teams of teams you are in. I think this inconsistency should be resolved, probably by notifying child teams as well as members directly assigned to the parent team.

[Edit: The "real-time alerts" feature for "Your team is assigned a review" seems to also correctly include child teams, so this seems just to affect the scheduled reminders.]

**Comment by mateus** [comment]
I wanted to setup reminders only for the child team, but it ends up spamming everyone from the parent team too. This limits the usage of this feature quite a bit.
